mnswaleh/apisendIT
# #161911732 signin endpoint

#### What does this PR do?
 Enables users to signin

#### Description of Task to be completed?
should have:
1. endpoint test
2. model function to verify user details
3. signin endpoint

#### How should this be manually tested?
1. on postman enter url, http://127.0.0.1:5000/api/v1/users/signin
2. send a post request with username and password
3. if user details add up you should receive user details

#### What are the relevant pivotal tracker stories?
#161911732

#### Screenshots (if appropriate)
![screenshot from 2018-11-13 09-39-47](https://user-images.githubusercontent.com/39084014/48396064-42ca5880-e72a-11e8-8d20-6f38e5d814a5.png)
![screenshot from 2018-11-13 09-38-17](https://user-images.githubusercontent.com/39084014/48396072-4827a300-e72a-11e8-93eb-70cb752d9388.png)
